docs/modules/commons/nav.adoc

```diff
@@ -3,3 +3,5 @@
 * xref:expressions.adoc[Expressions]
 * xref:vividus-steps.adoc[Steps]
 * xref:table-transformers.adoc[Table Transformers]
+* xref:http-configuration.adoc[HTTP Configuration]
+* xref:jira-configuration.adoc[JIRA Configuration]
```

Comment:
we don't have Jira client as a part of Vividus, so we shouldn't have such section